snowflake/table_snowflake_view_grant.go

```diff
@@ -31,7 +31,7 @@ func tableSnowflakeViewGrant(_ context.Context) *plugin.Table {
 			{Name: "schema_name", Type: proto.ColumnType_STRING, Transform: transform.FromQual("schema_name"), Description: "The name of the schema in which the view exists."},
 			{Name: "privilege", Type: proto.ColumnType_STRING, Description: "A defined level of access to an object."},
 			{Name: "created_on", Type: proto.ColumnType_TIMESTAMP, Description: "Date and time privilege was granted."},
-			{Name: "grant_option", Type: proto.ColumnType_BOOL, Description: "If set to TRUE, the recipient role can grant the privilege to other roles."},
+			{Name: "grant_option", Type: proto.ColumnType_STRING, Description: "If set to TRUE, the recipient role can grant the privilege to other roles."},
```

Comment:
The value suggests that it is a boolean column. Should we raise an issue in the SDK?